src/FlowDiagnostics.jl

```diff
@@ -15,9 +15,8 @@ using Oceananigans.Grids: Center, Face
 @inline fψ²(i, j, k, grid, f, ψ) = @inbounds f(i, j, k, grid, ψ)^2
 
 
-function RichardsonNumber(model; N²_bg=0, dUdz_bg=0, dVdz_bg=0)
+function RichardsonNumber(model; b=model.tracers.b, N²_bg=0, dUdz_bg=0, dVdz_bg=0)
```

Comment:
Oh, wow, I'm surprised this works. This sort of statement doesn't work in python :+1:

Reply:
Ah haha yeah I think it only works for kwargs but still pretty nice.